hands-on/launch-app.html.md.erb

```diff
@@ -73,7 +83,7 @@ app = "hellofly"
 ...
 ```
 
-The Fly CLI will always check to see if this file exists in the current directory, specifically for the `app` name value at the start. Many commands use this app name to determine which Fly App to apply to by default.
+The Fly CLI always checks to see if `fly.toml` exists in the current directory, specifically for the `app name` value at the start. Many commands use this app name to determine which Fly App to apply to by default.
```

Comment:
```suggestion
The Fly CLI always checks to see if `fly.toml` exists in the current directory, specifically for the `app` value at the start. Many commands use this app name to determine which Fly App to apply to by default.
```

(It is `app`:
```
app = "hello-gunicorn-flask2"
```
)

Reply:
OK! TY! I was thinking about the app name prompt, not the fly.toml file.

Reply:
@catflydotio So flyctl checks for a `fly.toml` file in the current directory, and then looks at the value of `app`? "At the start" refers to the start of the file?

Reply:
> So flyctl checks for a fly.toml file in the current directory, and then looks at the value of app? "At the start" refers to the start of the file?

That's right!